easylinks/src/main/webapp/navigation_script.js

```diff
@@ -96,7 +96,7 @@ function showDirectionsFromCurrentPosition(destLat, destLng) {
       (pos) => showDirectionsWhenSuccess(pos, destLat, destLng), 
       (error) => handleLocationError(false, error));
   } else {
-    handleLocationError(false, null);
+    handleLocationError(true, null);
```

Comment:
should this be `handleLocationError(false, null);` and the upper one be `handleLocationError(true, null);` ?
because if you get into the else block, it means that navigator.geolocation is not found, which implies the browser does not support geolocation.

Reply:
Done.